core/src/main/java/org/commonjava/maven/ext/core/ManipulationManager.java

```diff
@@ -82,11 +84,15 @@
 
     public static final String REPORT_JSON_OUTPUT_FILE = "reportJSONOutputFile";
 
+    // TODO: Config index value
```

Comment:
Chicken/egg ; can't fill this in till we do the docs and they are merged and then the index exists.